### PR TITLE
Mafia: Add information about roles and themes.

### PR DESCRIPTION
**Pull request by Snaquaza**

### CODE REVIEW

**Comment by HoeenCoder** [comment]
Will run the tests I promised to run later.

File: chat-plugins/mafia-data.js [hunk 1]
```diff
@@ -83,32 +83,1831 @@ exports.roles = Object.assign(Object.create(null), {
...
 		image: '<img width="75" height="75" src="//play.pokemonshowdown.com/fx/mafia-cop.png"/>',
 	},
+	coat: 'cop-of-all-trades',
+	cop_of_all_trades: 'cop-of-all-trades',
+	'cop-of-all-trades': {
+		name: 'Cop-of-all-Trades',
+		id: 'copofalltrades',
+		memo: ['Cop-of-all-Trades: You may use the actions of a Cop, a Seer, an FBI Agent, a Conspiracy Theorist, and a Private Investigator once a game each.'],
+	},
+	coroner: 'coroneresun',
```
Comment (whalemer) [inline chat-plugins/mafia-data.js]: coronereternalsun

File: chat-plugins/mafia-data.js [hunk 2]
```diff
@@ -83,32 +83,1831 @@ exports.roles = Object.assign(Object.create(null), {
...
+		id: 'parrotrole',
+		memo: ['Parrot Role: Each night, you can PM the host the name of a player. If they have an active ability, you use this ability on them.'],
+	},
+	politician: {
+		name: 'Politician',
+		id: 'politician',
+		memo: ['Politician: Each night, you can PM the host the name of two players. You give the vote of the former player to the latter player for the next day.'],
+	},
+	hooker: 'pretty_lady',
+	pl: 'prettylady',
```
Comment (whalemer) [inline chat-plugins/mafia-data.js]: pretty_lady

File: chat-plugins/mafia-data.js [hunk 3]
```diff
@@ -83,32 +83,1831 @@ exports.roles = Object.assign(Object.create(null), {
...
+			'Vanilla Townie',
+			'Vanilla Townie',
+			'Vanilla Townie',
+			'Vanilla Townie',
+			'Vanilla Townie',
+			'Vanilla Townie',
+			'Vanilla Townie',
+			'Doctor',
+			'Doctor',
+			'Coroner',
```
Comment (whalemer) [inline chat-plugins/mafia-data.js]: make this the ci variant

File: chat-plugins/mafia-data.js [hunk 4]
```diff
@@ -83,32 +83,1831 @@ exports.roles = Object.assign(Object.create(null), {
...
+	},
+	day: {
+		name: 'Day',
+		id: 'day',
+		memo: ['Day: You have to use your action during the Day.'],
+	},
+	death: {
+		name: 'Death',
+		id: 'death',
+		memo: 'Death: You can use your action upon dying.',
```
Comment (ZestOfLife) [inline chat-plugins/mafia-data.js]: memo of this isnt an array

**Comment by HoeenCoder** [comment]
Passes my tests, will make the modifications requested soon so this and other mafia PRs can progress.

**Pull request merged**